test/html/scrollToElement.html

```diff
@@ -0,0 +1,155 @@
+      <a id="second-a" href="#" onclick="scrollFunction('third');return false;">Click me to scroll to third element
+        <p id="second-clue"></p></a>
+    </div>
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    <p id="third">
+      <p id="third-clue"></p>
+      <a id="third-a" href="#" onclick="scrollFunction('fourth');return false;">Click me to scroll to fourth element</a>
+    </p>
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    <input id="fourth" type="text">
+    <p id="fourth-clue"></p>
+    <a id="fourth-a" href="#" onclick="scrollFunction('first');return false;">Click me to scroll to first element</a>
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+
+    <script>
+      function scrollFunction(id) {
+        let elmnt = document.getElementById(id);
```

Comment:
Since you're not changing the value of `elmnt` or the other variables after, you should use `const` to define these instead of `let`.